docs/public/installing-and-onfiguring-minikube-environment.adoc

```diff
@@ -0,0 +1,131 @@
+= Installing and configuring minikube environment
+
+[NOTE]
+----
+The guide only works with minikube configured with kubernetes 1.19.x or lower. `odo link` cannot link services successfully in a Kubernetes 1.20.x or newer environment.
+----
+
+It is recommended that users of this guide obtain a suitable system for running minikube with kubernetes. In practice this should be a 4 core system minimum. Before proceeding to the sample application, please follow the instructions for establishing a minikube environment:
+
+.Prerequisites
+* You have installed docker. See link:https://docs.docker.com/engine/install/[the Docker documentation].
```

Comment:
My concern with Docker is that it is leading to a lot of other steps to be performed further in the documentation. I know that the documentation I linked earlier had this info and you have used it to create this one. But I am inclined towards asking the users to create a VM using minikube. This way, we can probably skip a few steps and ask the users to refer the minikube documentation instead.

TL;DR - I'm very apprehensive of spinning up minikube using docker driver.

WDYT @kadel?